src/app/components/error.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+const Error = () => {
+  return (
+    <div className="flex min-h-screen flex-col items-center justify-center">
+      <h1 className="mb-20 text-4xl font-bold">404 Page</h1>
+      <div className="flex items-center gap-25">
+        <img src="/sadRice.svg" alt="sad Rice" className="w-xs" />
+        <div className="flex flex-col">
+          <p className="text-center text-2xl font-medium">
```

Comment:
```suggestion
          <p className="text-center text-2xl font-inter">
```

git pull origin dev to get the font